lib/sles4sap/ipaddr2.pm

```diff
@@ -236,6 +263,16 @@ sub ipaddr2_infra_deploy {
         name => $rg,
         region => $args{region});
 
+    # If image provided is a blob storage link, create image out of it
```

Comment:
Please create a UT variant about it in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/22_ipaddr2.t

Something like:
```
subtest '[ipaddr2_infra_deploy] with vhd' => sub {
```

where you simulate `ipaddr2_infra_deploy` to be called with `os => something.vhd`

This test can have the full `az_img_from_vhd_create` mocked or can mock `assert_script_run`

Reply:
Done, take a look